i.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+VERSION=$(curl --silent "https://api.github.com/repos/mytimecoin/pravda/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+OUT=pravda-$VERSION
+PACKAGE=PravdaSDK-$VERSION.tgz
+
+rm -rf /tmp/pravda-install
+mkdir /tmp/pravda-install
+cd /tmp/pravda-install
+
+echo "Downloading $PACKAGE..."
+curl -L -# -O --fail https://github.com/mytimecoin/pravda/releases/download/$VERSION/$PACKAGE -o $PACKAGE
+tar -xvzf $PACKAGE
+
+echo "Installing to /usr/local..."
+rm -rf $HOME/.pravda
+mv $OUT $HOME/.pravda
+printf '\nexport PATH="$HOME/.pravda/bin/:$PATH"' >> $HOME/.bash_profile
+source $HOME/.bash_profile
```

Comment:
hm... look at this question, please 
https://askubuntu.com/questions/121073/why-bash-profile-is-not-getting-sourced-when-opening-a-terminal?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa